src/rpc/fees.cpp

```diff
@@ -33,11 +33,11 @@ static RPCHelpMan estimatesmartfee()
         "in BIP 141 (witness data is discounted).\n",
         {
             {"conf_target", RPCArg::Type::NUM, RPCArg::Optional::NO, "Confirmation target in blocks (1 - 1008)"},
-            {"estimate_mode", RPCArg::Type::STR, RPCArg::Default{"conservative"}, "The fee estimate mode.\n"
-            "Whether to return a more conservative estimate which also satisfies\n"
-            "a longer history. A conservative estimate potentially returns a\n"
-            "higher feerate and is more likely to be sufficient for the desired\n"
-            "target, but is not as responsive to short term drops in the\n"
+            {"estimate_mode", RPCArg::Type::STR, RPCArg::Default{"economical"}, "The fee estimate mode.\n"
+            "Whether to return a more economical estimate\n"
+            "An economical estimate potentially returns a\n"
+            "lower feerate that is estimated to be sufficient for the desired\n"
+            "target, it is also a bit more responsive to short term drops in the\n"
```

Comment:
The old text here seems like a better explanation of why you would change this from the new default?

Reply:
I Added it to the PR description

Reply:
Not sure if you understood what I meant, apologies if you did already: I mean if I'm looking at the estimatesmartfee help, trying to figure out if I want to change from the (new) default of "economical", the existing explanation of what "conservative" does seems more helpful than an explanation of what the default option does. So just changing the default without changing the rest of the help text seems slightly better, doesn't it?

Reply:
I now understand thanks for clarifying.
yes  it does it will even be better to document the short description of both modes and indicate `economical` is the default ?

Reply:
I added a commit, that add more information about the modes.

On master b27afb7fb774809c9c075d56e44657e0b607a00c
This is the help output
```
2. estimate_mode    (string, optional, default="conservative") The fee estimate mode.
                    Whether to return a more conservative estimate which also satisfies
                    a longer history. A conservative estimate potentially returns a
                    higher feerate and is more likely to be sufficient for the desired
                    target, but is not as responsive to short term drops in the
                    prevailing fee market. Must be one of (case insensitive):
                    "unset"
                    "economical"
                    "conservative"
```

With this PR
```
2. estimate_mode    (string, optional, default="economical") The fee estimate mode.
                    "unset" 
                    no mode set
                    "economical" 
                    Economical estimates potentially return a lower
                    feerate that is estimated to be sufficient for the
                    desired target. It is also more responsive to
                    short-term drops in the prevailing fee market.
                    "conservative" 
                    Conservative estimates satisfy a longer history.
                    A conservative estimate potentially returns a higher
                    feerate and is more likely to be sufficient for
                    the desired target, but is not as responsive
                    to short-term drops in the prevailing fee market.
                    
```

The first commit https://github.com/bitcoin/bitcoin/pull/30275/commits/2ee3d774815f5e1840cb37d2a92007f19c217e4c doc change change affect other RPC's that require estimation mode.

Reply:
Maybe for this PR, it would be cleaner to drop the first commit and just change the default, so we can focus the review on the setting instead of the help text?

2ee3d774815f5e1840cb37d2a92007f19c217e4c seems helpful but can be a separate PR.

Reply:
I dropped the first commit and use default help text as @ajtowns  suggested.
 
---
Yes I think it will be beneficial to improve the documentation
I will open up https://github.com/bitcoin/bitcoin/commit/2ee3d774815f5e1840cb37d2a92007f19c217e4c in a separate pull.

Thanks _all_ for your reviews.

Reply:
I agree that updating the help text (in a followup) would be nice. I think (a slightly modified version of) the explanation from the 0.15.0 release notes would be a good improvement, as IMO it's clearer than wording like "is more likely to be sufficient for the desired target":

```
2. estimate_mode    (string, optional, default="economical") The fee estimate mode.
                    *Conservative* estimates use longer time horizons to produce an estimate which
                    is less susceptible to rapid changes in fee conditions. *Economical* estimates
                    use shorter time horizons and will be more affected by short-term changes in
                    fee conditions. Economical estimates may be considerably lower during periods
                    of low transaction activity but may result in transactions being unconfirmed if
                    prevailing fees increase rapidly.
```